crates/genesis/src/lib.rs

```diff
@@ -28,28 +28,31 @@ use std::collections::HashMap;
 
 /// The genesis block specification.
 #[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq, Eq)]
-#[serde(rename_all = "camelCase", default)]
+#[serde(rename_all = "camelCase")]
```

Comment:
I only notice this now,

I really don't understand why it is necessary to remove the struct's default and put it on every field